.github/assets/kurtosis_network_params.yaml

```diff
@@ -0,0 +1,15 @@
+participants:
+  - el_type: geth
```

Comment:
just out of curiosity: why do we need geth here?

Reply:
I think so that we have something to compare against in case there's a fork. Otherwise we wouldn't know that our execution result doesn't match with other clients.

Reply:
yep exactly, ideally we should run tests on a testnet as similar to prod as possible, using diverse clients will ensure that reth interoperates with them properly.

this initial setup with just two client pairs ia just for validatung the config, imo we should add more if the CI workers allow for that